monai/networks/blocks/convolutions.py

```diff
@@ -119,10 +112,9 @@ def __init__(
         is_transposed: bool = False,
         padding: Optional[Union[Sequence[int], int]] = None,
         output_padding: Optional[Union[Sequence[int], int]] = None,
-        dimensions: Optional[int] = None,
     ) -> None:
         super().__init__()
-        self.dimensions = spatial_dims if dimensions is None else dimensions
+        self.dimensions = spatial_dims
```

Comment:
Should we change `self.dimensions` to `self.spatial_dims` as well or are there too many problems with that?